pycbc/inference/sampler/multinest.py

```diff
@@ -396,4 +397,9 @@ def from_config(cls, cp, model, nprocesses=1, use_mpi=False):
                            v is not None}
         obj = cls(model, nlivepoints, constraints=constraints,
                   **optional_kwargs)
+        obj.setup_output(obj, output_file)
+        if not obj.new_checkpoint:
+            obj.resume_from_checkpoint(cp)
```

Comment:
There doesn't seem to be a `resume_from_checkpoint` implementation in the multinest sampler like you've put in all the others. I think this will fail because the abstract method hasn't been overridden.